config/prow/cluster/oauth2-proxy/helm/values.yaml

```diff
@@ -0,0 +1,67 @@
+# Check values.yaml of oauth2-proxy for help https://github.com/oauth2-proxy/manifests/blob/main/helm/oauth2-proxy/values.yaml
+extraArgs:
+  whitelist-domain: .prow.gardener.cloud
+  cookie-domain: .prow.gardener.cloud
+  provider: github
+  github-org: gardener
```

Comment:
Does that mean, only `gardener` org members will be allowed to log in to prow's monitoring? 😢

Reply:
Not anymore 😄